README.md

```diff
@@ -149,15 +153,15 @@ several other libraries, including:
 
 ## Supported Rust Versions
 
-Tokio is built against the latest stable, nightly, and beta Rust releases. The
-minimum version supported is the stable release from three months before the
-current stable release version. For example, if the latest stable Rust is 1.29,
-the minimum version supported is 1.26. The current Tokio version is not
-guaranteed to build on Rust versions earlier than the minimum supported version.
+Tokio is built against the latest stable release. The minimum supported version is 1.39.
+The current Tokio version is not guaranteed to build on Rust versions earlier than the
+minimum supported version.
```

Comment:
Personally, I'd prefer to leaving "at least three months" as the MSRV policy and also mentioning the current MSRV. ([like crossbeam](https://github.com/crossbeam-rs/crossbeam#compatibility))

Reply:
I'm OK w/ setting it to 1.39 for 0.2. We should have 0.3 "soonish". We probably should revise for 1.0